lib/meta_tags/meta_tags_collection.rb

```diff
@@ -162,6 +162,11 @@ def extract_robots
         calculate_robots_attributes(result, attributes)
       end
 
+      robots = extract(:robots).presence
+      if robots
+        result['robots'].concat(robots.map { |k, v| "#{k}:#{v}" })
+      end
```

Comment:
Wonder what would be the expected behavior for cases when a specific bot is configured, like

```ruby
set_meta_tags(nofollow: 'googlebot', robots: 'nosnippet')
```

Or, for example, how to configure `nosnippet` for one bot, but no the others...

Reply:
Interesting question that I didn't think about. 
For that case, how about just `set_meta_tags(googlebot: 'nosnippet')`?

To implement this, I would tweak `extract_robots` and check for hash keys `/[google|bing]bot/`, the only two that I found support nosnippet and other snippet-related configs.